src/frontend/apps/impress/src/features/docs/doc-export/components/ModalExport.tsx

```diff
@@ -109,10 +114,12 @@ export const ModalExport = ({ onClose, doc }: ModalExportProps) => {
         resolveFileUrl: async (url) => exportCorsResolveFileUrl(doc.id, url),
       });
 
-      blobExport = await exporter.toBlob(exportDocument);
+      const originalBlob = await exporter.toBlob(exportDocument);
```

Comment:
I don't think you need to do this zip trick, try that:
```suggestion
      blobExport = await exporter.toBlob(exportDocument, {
        documentOptions: { title: documentTitle },
        sectionOptions: {},
      });
```

Reply:
Oh, this is way much simpler and I can get rid of addTitleToDocx function